assets/i18n/strings_tr.i18n.json

```diff
@@ -0,0 +1,196 @@
+{
+    "locale": "Türkçe",
+    "appName": "LocalSend",
+    "general": {
+      "accept": "Kabul et",
+      "add": "Ekle",
+      "advanced": "Gelişmiş",
+      "cancel": "İptal et",
+      "close": "Kapat",
+      "confirm": "Onayla",
+      "continueStr": "Devam et",
+      "copy": "Kopyala",
+      "copiedToClipboard": "Panoya kopyalandı",
+      "decline": "Reddet",
+      "done": "Bitir",
+      "edit": "Düzenle",
+      "error": "Hata",
+      "example": "Örnek",
+      "files": "Dosyalar",
+      "finished": "Sonlandırıldı",
+      "hide": "Gizle",
+      "off": "Kapalı",
+      "offline": "Çevrim dışı",
+      "on": "Açık",
+      "online": "Çevrim içi",
+      "open": "Aç",
+      "queue": "Sıra",
+      "quickSave": "Hızlı Kaydetme",
+      "renamed": "Yeniden Adlandırıldı",
+      "reset": "Sıfırla",
+      "restart": "Yeniden Başlat",
```

Comment:
In some places you capitalized, in others you did not. It will be more efficient if they solve this situation in css. i think we will must use a standard spelling for binary words.

Reply:
In general ok, the translations can be taken in if we be attention to the comment I wrote above.

Reply:
Yes, I agree with you. I'll fixing them and open a new pr again. Thanks for your review. 👏 
Hope we can see the turkish version of localsend asap.